tests/Fixtures/e2e/Test_Frameworks/phpunit.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<phpunit backupGlobals="false"
+         backupStaticAttributes="false"
+         bootstrap="./vendor/autoload.php"
```

Comment:
`vendor/autoload.php` (all relative paths are necessarily relative to the config file)